docs/ddl-of-carbondata.md

```diff
@@ -681,24 +682,24 @@ Users can specify which columns to include and exclude for local dictionary gene
 
      **NOTE:** Drop Complex child column is not supported.
 
-   - ##### CHANGE DATA TYPE
+   - ##### CHANGE COLUMN NAME/TYPE
    
-     This command is used to change the data type from INT to BIGINT or decimal precision from lower to higher.
+     This command is used to change the column's name and the data type from INT to BIGINT or decimal precision from lower to higher and rename column.
```

Comment:
Write this as below
`This command is used to change column name and the data type from INT to BIGINT or decimal precision from lower to higher.`